.config/smtp.config.php

```diff
@@ -11,5 +11,9 @@
     'mail_smtppassword' => getenv('SMTP_PASSWORD') ?: '',
     'mail_from_address' => getenv('MAIL_FROM_ADDRESS'),
     'mail_domain' => getenv('MAIL_DOMAIN'),
+    'mail_smtpdebug' =>  getenv('SMTP_DEBUG') ?: false,
+    'mail_template_class' =>  getenv('SMTP_TEMPLATE_CLASS') ?: '\OC\Mail\EMailTemplate',
```

Comment:
```suggestion
    'mail_template_class' =>  getenv('SMTP_TEMPLATE_CLASS') ?: '',
```

Please use `''` as default: https://github.com/nextcloud/server/blob/5bf3d1bb384da56adbf205752be8f840aac3b0c5/lib/private/Mail/Mailer.php#L137